cmd/analyzer/parse_args.go

```diff
@@ -180,9 +191,48 @@ func ParseInArgs(cmdlineArgs []string) (*InArgs, error) {
 	if err != nil {
 		return nil, err
 	}
+	err = parseConnectionDescription(args.ConnectionDescription, &args)
+	if err != nil {
+		return nil, err
+	}
 
 	return &args, nil
 }
+
+func parseConnectionDescription(connectionDescription *string, args *InArgs) error {
+	if *connectionDescription == "" {
```

Comment:
This is not needed. We will soon add `AnalysisType` for query, and then allow specifying these parms only when the mode will be query.
In the meantime just have validity checks that if protocol is specified is legal and that if src/dst min/max ports are specified they are withing range, and that if both min and max are specified then the min is lower than or equal to max